docs/BuildExtensibility.md

```diff
@@ -0,0 +1,141 @@
+![UI5 icon](https://raw.githubusercontent.com/SAP/ui5-tooling/master/docs/images/UI5_logo_wide.png)
+
+# UI5 Build Extensibility
+
+The UI5 Tooling enables it to define a custom build for your UI5 project by adding additional tasks to the  [standard task list](https://github.com/SAP/ui5-builder/blob/master/README.md#tasks).
+
+## Configuration
+
+The additional build tasks can be defined in the project configuration within the `ui5.yaml` file.
```

Comment:
Add link to ui5-project README configuration chapter

Reply:
Done